kili/types.py

```diff
@@ -269,6 +269,7 @@ class Asset:
     jsonMetadata = 'jsonMetadata'
     labels = LabelWithoutLabelOf
     latestLabel = LabelWithoutLabelOf
+    latestLabelCompute = LabelWithoutLabelOf
```

Comment:
Shouldn't it be `latestLabelCompute = 'latestLabelCompute'`?

Reply:
Nope because it returns a label